spec.bs

```diff
@@ -1342,9 +1356,9 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
   1. Let |topLevelDirectFromSellerSignalsRetrieved| be false.
   1. [=list/For each=] |component| in |auctionConfig|'s [=auction config/component auctions=],
     [=parallel queue/enqueue steps|enqueue the following steps=] to |queue|:
-    1. Let |compWinner| be the result of running [=generate and score bids=] with |component|,
-      |auctionConfig|, |global|, and |topLevelOrigin|.
-    1. If |compWinner| is failure, return failure.
+    1. Let |compWinnerInfo| be the result of running [=generate and score bids=] with |component|,

```

Comment:
(to helper reviewer understand why making this change):
runAdAuction() expects a [=leading bid info=], instead of a [=generated bid=].

Reply:
@MattMenke2 maybe renaming "leading bid" to "generated bid", or "leading generated bid"? Let me know if it helps. If so, I can rename them in a separate PR.